lib/ansible/modules/crypto/entrust/ecs_certificate.py

```diff
@@ -0,0 +1,936 @@
+                description: Custom dropdown field.
+                type: str
+            dropdown3:
+                description: Custom dropdown field.
+                type: str
+            dropdown4:
+                description: Custom dropdown field.
+                type: str
+            dropdown5:
+                description: Custom dropdown field.
+                type: str
+    cert_expiry:
+        description:
+            - The date the certificate should be set to expire, as an RFC3339 compliant date or date-time. For example,
+              C(2020-02-23), C(2020-02-23T15:00:00.05Z).
+            - I(cert_expiry) is only supported for requests of C(request_type=new) or C(request_type=renew). If C(request_type=reissue),
+              I(cert_expiry) will be used for the first certificate issuance, but subsequent issuances will have the same expiry as the initial
+              certificate.
+            - A reissued certificate will always have the same expiry as the original certificate.
+            - Note that only the date (day, month, year) is supported for specifying expiry date. If you choose to specify an expiry time with the expiry date,
+              the time will be adjusted to Eastern Standard Time (EST). This could have the unintended effect of moving your expiry date to the previous day.
+            - Applies only to accounts with a pooling inventory model.
+            - Only one of I(cert_expiry) or I(cert_lifetime) may be specified.
+        type: str
+    cert_lifetime:
+        description:
+            - The lifetime of the certificate.
+            - Applies to all certificates for accounts with a non-pooling inventory model.
+            - I(cert_lifetime) is only supported for requests of C(request_type=new) or C(request_type=renew). If C(request_type=reissue), I(cert_lifetime) will
+              be used for the first certificate issuance, but subsequent issuances will have the same expiry as the initial certificate.
+            - Applies to certificates of I(cert_type)=C(CDS_INDIVIDUAL, CDS_GROUP, CDS_ENT_LITE, CDS_ENT_PRO, SMIME_ENT) for accounts with a pooling inventory
+              model.
+            - C(P1Y) is a certificate with a 1 year lifetime.
+            - C(P2Y) is a certificate with a 2 year lifetime.
+            - C(P3Y) is a certificate with a 3 year lifetime.
+            - Only one of I(cert_expiry) or I(cert_lifetime) may be specified.
+        type: str
+        choices: [ P1Y, P2Y, P3Y ]
+seealso:
+    - module: openssl_privatekey
+      description: Can be used to create private keys (both for certificates and accounts).
+    - module: openssl_csr
+      description: Can be used to create a Certificate Signing Request (CSR).
+extends_documentation_fragment:
+    - ecs_credential
+'''
+
+EXAMPLES = r'''
+- name: Request a new certificate from Entrust with bare minimum parameters. Will request a new certificate if current one is valid but within 30 days of
```

Comment:
```suggestion
- name: Request a new certificate from Entrust with bare minimum parameters.
        Will request a new certificate if current one is valid but within 30 days of
```
These lines won't be broken automatically, so if they are too long, the user has to scroll horizontally to see the complete line.

(Same for other `name:` fields below.)

Reply:
I did break them up to hit the 160 character limit, is reducing to, say, <120 fine?

Reply:
I would even aim for 80. It depends a lot on what kind of device you use to read the docs. See for example https://docs.ansible.com/ansible/latest/modules/openssl_certificate_module.html#examples There is one line "`- name: Check the new certificate again for validity with the same parameters, this time failing the play if it is still invalid`" with 128 characters, and that already requires horizontal scrolling on my laptop, where the browser-window is full width (I can't see the last 12 characters).